debian/control

```diff
@@ -0,0 +1,14 @@
+Source: sio2jail
+Section: utils
+Priority: optional
+Maintainer: Michał Sidor <michcioperz@asirpa.iscute.ovh>
+Build-Depends: debhelper (>= 11), cmake
```

Comment:
you don't have `-DWITH_DOCS=NO`, so this should need scdoc, right? Or is it using the vendored version?